nodejs/eks/nodegroup.ts

```diff
@@ -1770,12 +1770,18 @@ function createMNGCustomLaunchTemplate(
                 args.clusterName,
             ])
             .apply(([clusterName, clusterEndpoint, clusterCertAuthority, argsClusterName]) => {
-                return `#!/bin/bash
+                return `MIME-Version: 1.0
+Content-Type: multipart/mixed; boundary="==MYBOUNDARY=="
+
+--==MYBOUNDARY==
+Content-Type: text/x-shellscript; charset="us-ascii"
+
+#!/bin/bash
 
-            /etc/eks/bootstrap.sh --apiserver-endpoint "${clusterEndpoint}" --b64-cluster-ca "${clusterCertAuthority}" "${
+/etc/eks/bootstrap.sh --apiserver-endpoint "${clusterEndpoint}" --b64-cluster-ca "${clusterCertAuthority}" "${
                     argsClusterName || clusterName
                 }"${bootstrapExtraArgs}
```

Comment:
Seems like these lines should be unindented.

Reply:
These lines were indented from the linter. Since they're also typescript variables, the final text will be formatted as a single line.